ui/src/app/cohort-review/status-filter/status-filter.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="form-group">
+  <label for="option-status-picker">
+    Select Status(es)
+  </label>
+  <div class="select multiple">
+    <select id="option-status-picker" multiple [formControl]="selection">
```

Comment:
So you can iterate your CohortStatus enum like so: https://stackoverflow.com/questions/38554562/how-can-i-use-ngfor-to-iterate-over-typescript-enum-as-an-array-of-strings